sdks/builds/ios.mk

```diff
@@ -32,18 +32,20 @@ _ios_LDFLAGS=
 #  $(3): arch (arm or aarch64)
 #
 # Flags:
-#  ios_$(1)_AC_VARS
-#  ios_$(1)_CFLAGS
-#  ios_$(1)_CPPFLAGS
-#  ios_$(1)_CXXFLAGS
-#  ios_$(1)_LDFLAGS
-#  ios_$(1)_BITCODE_MARKER
+#  ios_$(_$(1)_normalized)_AC_VARS
```

Comment:
Whats the point of these changes ? Why do we have to make these variables uppercase ?

Reply:
I pushed the wrong version :)